plugins/communicators/ssh/communicator.rb

```diff
@@ -21,6 +21,7 @@ module CommunicatorSSH
     class Communicator < Vagrant.plugin("2", :communicator)
       PTY_DELIM_START = "bccbb768c119429488cfd109aacea6b5-pty"
       PTY_DELIM_END = "bccbb768c119429488cfd109aacea6b5-pty"
+      CMD_GARBAGE_MARKER = SecureRandom.uuid.freeze
```

Comment:
I think we might want to actually hard code this just like we do the PTY delimiters. As written, each vagrant invocation will get a different UUID, which might not be desirable, especially for debugging.